server.js

```diff
@@ -16,10 +19,38 @@ app.use(ejsLayouts);
 app.use(require('morgan')('dev'));
 
 // Routes
+
+
+app.get('/', (req, res)=>{
+  let qs = {
+    params:{
+      s: 'movie',
+      apikey: API_KEY
+    }
+  }
+    axios.get('http://www.omdbapi.com', qs)
+    .then(function(response){
+      console.log(response.data);
+      let movie = reponse.data.Search
+      res.render('home', {movie})
```

Comment:
the res.render() is going to render a .ejs file that is in your views folder.  You don't have a home.ejs file, but you do have an index.ejs file.  This line on 35 should render the index.ejs file.  How would you update what you have to get it working?